apps/meteor/app/livechat/server/hooks/saveLastMessageToInquiry.ts

```diff
@@ -8,12 +8,7 @@ import { RoutingManager } from '../lib/RoutingManager';
 callbacks.add(
 	'afterSaveMessage',
 	(message, room) => {
-		if (!isOmnichannelRoom(room)) {
-			return message;
-		}
-
-		// skip callback if message was edited
-		if (isEditedMessage(message)) {
+		if (!isOmnichannelRoom(room) || isEditedMessage(message) || message.t) {
```

Comment:
Can you explain me why this was needed? 👀 I mean, to combine the ifs, as the only new condition i see is "message.t"

Reply:
Previously, the transfer system message was getting stored within the lastMessage prop which is y it was showing the message text as `undefined`. This new condition is added so it can filter out messages like that which sidebar is not capable of rendering